.changeset/lovely-swans-repeat.md

```diff
@@ -0,0 +1,10 @@
+---
+'@shopify/shopify-app-remix': minor
+---
+
+- Internally rearranged source files to create a better separation between backend and frontend code, so we can add frontend-specific exports in the future.
+- A new export path `@shopify/shopify-app-remix/react` will now contain those frontend exports.
```

Comment:
Would only ship this change once we have something to put in the react export?  I'm in two minds here.  Shipping often vs shipping a large changelog that doesn't contain much the partner needs to action.

Reply:
I have a different PR coming with the `AppProvider` that will be exported out of this path. We can fix up the changelog once everything is combined to ensure it's reasonable.